### DISCUSSION TITLE
1.0.0-beta.13

### DISCUSSION THREAD

**Discussion by mrkai77** [mrkai77/Loop#143] [Announcements]
## 📰 Important News
* 🎉 Loop now has a GitHub Sponsors page, [check it out](https://github.com/sponsors/MrKai77) and support the project!
* ⚠️ This new version will unfortunately get rid of all your current keybindings. However, to help prevent this in the future, you can now export a copy of your keybinds!

## ✨ New Features
* #74: Ability to hide Loop until a direction is chosen (@MrKai77)
* #78: Option to hide Loop's menubar item (@MrKai77)
* #87: Restore initial frame when window is dragged (@MrKai77)
* #93: Option to "Almost Maximize" a window (@MrKai77)
* #96: Import/Export settings (@MrKai77)
* #98: Custom window sizes & custom cycle keybinds! (@MrKai77)
* #101: Respect Stage Manager (@MrKai77)
* #105: Copy Button for Version Number (@MrKai77)
* #106: Resize window under cursor (@MrKai77)
* #113: Add link to GitHub repository in settings window (@MrKai77)
* #115: Remove initial flicker when window animations are enabled (@MrKai77)
* #128: Move Windows in real-time "Show Preview when looping" is disabled (@MrKai77)
* #139: Remove background grid in settings window (@MrKai77)

## 🐛 Bug Fixes
* #104: Settings window is duplicated instead of focused (@MrKai77)
* #127: "Double-click trigger key to trigger Loop" doesn't work (@MrKai77)

<hr /><em>This discussion was created from the release <a href='https://github.com/MrKai77/Loop/releases/tag/1.0.0-beta.13'>1.0.0-beta.13</a>.</em>

**Comment by Dhairy2003** [comment]
Can't thank u enough for this update , thanks for adding the custom cycle. 🥹

**Comment by Chris-Asaka** [comment]
Hi, I just tried installing Loop and received an error post-install related to the macOS I am running, macOS Sonoma Version 14.2.1.  When will a version be available for macOS Version 14.x?

“Loop.app” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

**Reply by ankidoc** [reply]
It works in Sonoma. Just go to the app's save location (or first drag it into applications folder), right click while holding down the cmd key, and click open. Should do the trick.

**Comment by Dhairy2003** [comment]
Move it to ur application folder and then right click open , don't use
double click to open it
